threejs/fxhash_Rooms/src/index.js

```diff
@@ -114,8 +124,11 @@ for (let i = 0; i < figures.length; i++) {
       roomShape,
       figures[0].extrudeSettings
     );
+  } else if (figures[i].geometry.type === "PlaneGeometry") {
+    groundPlaneGeometry = new THREE[figures[i].geometry.type](
+      ...figures[i].geometry.args
+    );
```

Comment:
@Anaglyph-ic seems I'm missing something. When I add the planeGeometry into the scene I get errors with the figures not being attributed correctly. Maybe the way I'm creating and adding them is wrong? Can you take a look when you have a moment?